docker/importer/importer.py

```diff
@@ -300,7 +300,23 @@ def _process_updates_bucket(self, source_repo):
 
       logging.info('Bucket entry triggered for for %s/%s', source_repo.bucket,
                    blob.name)
-      original_sha256 = osv.sha256_bytes(blob.download_as_bytes())
+      blob_bytes = blob.download_as_bytes()
+
+      # Might be an array, might not
+      vulnerabilities = json.loads(blob_bytes)
+
+      if not isinstance(vulnerabilities, list):
+        vulnerabilities = [vulnerabilities]
+
+      def identical_vuln_exist(vuln):
+        bug = osv.Bug.get_by_id(vuln['id'])
+        return bug and bug.import_last_modified == datetime.datetime.fromisoformat(
+            vuln['modified'].replace('Z', '+00:00'))
```

Comment:
You can avoid this hack if you use `osv.parse_vulnerabilities()` which properly handles these. 

You just need to call `vuln.modified.ToDatetime()`